CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+* Return `500` error response instead of exceptions when `raise_app_exceptions=False` is set on `ASGITransport`. (#2669)
```

Comment:
Looks good, tho needs to be under a new `## development` header, instead of included in the 0.24.1 release notes.

Reply:
Oh, my bad, sorry